submissions/ogennaOgbogu/htmlForms/contactUs.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Kindly format this page properly. Check out this [link](https://google.github.io/styleguide/htmlcssguide.html) to help you in styling your HTML and CSS pages.

Reply:
Hi @heisbabz, I understand all the other changes except this one. Please explain what you want me to do, I'm confused.

Reply:
What am saying is that you should format it properly. The link I shared with you is the HTML/CSS style guide.

Reply:
I have read the guide, I still do not understand what you want me to do, so if you be so inclined, kindly explain exactly want you want me to do.